src/System.Net.Http/src/System/Net/Http/Headers/HttpHeaderValueCollection.cs

```diff
@@ -201,18 +201,18 @@ Collections.IEnumerator Collections.IEnumerable.GetEnumerator()
 
         #endregion
 
-        public override string ToString()
+	internal string GetHeaderStringWithoutSpecial()
         {
-            return _store.GetHeaderString(_headerName);
-        }
+	    if (!IsSpecialValueSet)
+	    {
+		return ToString();
+	    }
+	    return _store.GetHeaderString(_headerName, _specialValue);
+	}
 
-        internal string GetHeaderStringWithoutSpecial()
+        public override string ToString()
         {
-            if (!IsSpecialValueSet)
-            {
-                return ToString();
-            }
-            return _store.GetHeaderString(_headerName, _specialValue);
+            return _store.GetHeaderString(_headerName);
         }
```

Comment:
Why were these swapped?  These kinds of changes mixed in with dead code removal makes it much harder to understand the diff.

Also, it looks like the formatting / whitespace was messed up, with tabs replacing spaces in some cases.